docroot/themes/custom/uids_base/scss/components/card.scss

```diff
@@ -49,6 +49,17 @@
 }
 
 
+.card--centered {
+  .fa-li {
+    left: 0;
+    position: relative;
+  }
+
+  .fa-ul {
+    margin-left: 0;
+  }
+}
+
 // Remove padding if no section background and bg--white is set on card.
 .layout__container:not([class*="bg--"]),
 .view-display-id-page_articles:not([class*="bg--"]) {
```

Comment:
It seems like we would not want this article-specific class in our card.scss file.

Reply:
Yes, my goal currently is to reduce the amount of extra CSS files for the articles lists. Then I will further refactor this file to move items elsewhere.

Reply:
Got it!